### PR TITLE
feat(background-services): add CLI tooling for listing, running, and crontab generation

### LINKED ISSUE #11328: feat(background-services): Add CLI tooling for listing, running, and crontab generation
**Parent:** #11325 (step 3 of 5)
**Depends on:** Step 2 (BackgroundServiceRunner extraction)

## Problem

A sysadmin who wants to run background services via cron must:
1. Query the `background_services` table directly to discover service names
2. Know the undocumented CLI calling convention: `php library/ajax/execute_background_services.php default phimail 0`
3. Manually translate `execute_interval` values into cron expressions

There is no self-documenting CLI interface.

## Proposed Change

Add a CLI entry point at `bin/background-services` with three subcommands:

### `list` — Show registered services

```bash
bin/background-services list [--site=default]

# NAME               TITLE                              ACTIVE  INTERVAL  LAST RUN
# phimail            phiMail Direct Messaging Service    yes     5 min     2026-03-28 10:15:00
# Email_Service      Email Service                       yes     2 min     2026-03-28 10:18:00
# UUID_Service       UUID Creation Service               yes     240 min   2026-03-28 06:00:00
# X12_SFTP           SFTP Claims to X12 Partner          no      1 min     never
# MedEx              MedEx Messaging Service             no      manual    never
```

### `run` — Execute a specific service

```bash
bin/background-services run <name> [--site=default] [--force]
```

Delegates to `BackgroundServiceRunner::run()`. Exit code reflects outcome (0 = ran, 1 = error, 2 = skipped/not due).

### `crontab` — Generate crontab entries

```bash
bin/background-services crontab [--site=default] [--php=/usr/bin/php]

# # OpenEMR Background Services (site: default)
# */5 * * * *   /usr/bin/php /path/to/bin/background-services run phimail --site=default
# */2 * * * *   /usr/bin/php /path/to/bin/background-services run Email_Service --site=default
# 0 */4 * * *   /usr/bin/php /path/to/bin/background-services run UUID_Service --site=default
```

Translates `execute_interval` (minutes) into reasonable cron expressions. Output is a starting point — sysadmins adjust as needed.

## Backward Compatibility

The old CLI interface (`php library/ajax/execute_background_services.php default phimail 0`) continues to work unchanged.

## Files

- `bin/background-services` — new CLI entry point
- Uses `BackgroundServiceRunner` from step 2
- Pattern reference: `library/allow_cronjobs.php` (existing CLI bootstrap helper)

### PR DESCRIPTION
**Pull request by kojiromike**
## Summary

Adds `BackgroundServicesCommand` (`php bin/console background:services`) with three subcommands:

- **`list`** — displays registered services in a formatted table (name, title, active, running, interval, next run)
- **`run --name=X [--force]`** — executes a specific service via `BackgroundServiceRunner`, with meaningful exit codes
- **`crontab [--php=/path/to/php]`** — generates crontab entries from active service intervals

Key design decisions:
- Uses `BackgroundServicesRow` type stub with `(int)` casts for ADOdb `numeric-string` values
- `minutesToCron()` only emits intervals representable as valid cron entries (minutes dividing 60, hours dividing 24, exactly 1 day); all others get a `# cannot represent` comment
- Shell-escapes paths in crontab output via `escapeshellarg()`
- Uses `$this->getGlobalsBag()` (injected via `IGlobalsAware`) instead of singleton
- Auto-discovered by `SymfonyCommandRunner`; old CLI interface unchanged

## Test plan

- [x] `composer phpunit-isolated` — 14 tests pass (list, run, crontab, edge cases)
- [x] `composer phpstan` — no errors
- [x] Pre-commit hooks pass
- [x] Docker: `docker compose exec openemr php bin/console background:services list` shows registered services
- [x] Docker: `docker compose exec openemr php bin/console background:services crontab` generates valid cron entries

Closes #11328

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: src/Common/Command/BackgroundServicesCommand.php [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    {
+        $this
+            ->setName('background:services')
+            ->setDescription('List, run, or generate crontab entries for background services')
+            ->setDefinition(
+                new InputDefinition([
+                    new InputArgument('action', InputArgument::REQUIRED, 'Action to perform: list, run, or crontab'),
+                    new InputOption('name', null, InputOption::VALUE_REQUIRED, 'Service name (required for "run")'),
+                    new InputOption('force', 'f', InputOption::VALUE_NONE, 'Bypass interval check (for "run")'),
+                    new InputOption('php', null, InputOption::VALUE_REQUIRED, 'PHP binary path (for "crontab")', PHP_BINARY),
```
[the remark this answers is not in the document]
Comment (kojiromike) [reply src/Common/Command/BackgroundServicesCommand.php:49]: Not needed. bin/console already parses the site option globally before dispatching to the command.
